add instruction for monitor renotification (#10546)
DataDog/documentation: content/en/monitors/notifications.md

### Renotify

Enable monitor renotification (optional), which is useful to remind your team a problem is not solved. 
If enabled, you are given the option to include an escalation message sent any time the monitor remain in the `alert` or `no data` state for the configured time.

> The escalation message can be added in two ways:
> 
> * In the escalation_message box (UI) or attribute (API)
> * (recommended) By leveraging the `{{#is_renotify}}` block in the original message

Reviewer: ```suggestion
The escalation message can be added in the following ways:

* In the `{{#is_renotify}}` block in the original notification message (recommended).
* In the *escalation_message* field in the app.
* With the `escalation_message` attribute in the API.
```

> The original notification message is always included therefore you want to leverage the `{{#is_renotify}}` block in the following conditions:

Reviewer: ```suggestion
If you use the `{{#is_renotify}}` block, keep in mind that the original notification message is also included in the renotification, so:
```

> 1. The escalation message is only there to add some extra details and should not contain the original message details
> 2. The escalation message should only be sent for a subset of groups

Reviewer: ```suggestion
1. Include only extra details in the `{{#is_renotify}}` block and don't repeat the original message details.
2. Send the escalation message to a subset of groups.
```
Is "to a subset of groups" correct? I wasn't sure what sending a message "for" a subset meant, but maybe it was correct.

{{/is_exact_match}}
```

{{% /tab %}}
{{% tab "is_renotify" %}}

To send an escalation message to a different destination just for the `production` environment:

```text
{{#is_renotify}}
{{#is_match "env" "production"}}
  This is an escalation message sent to @dev-team@company.com
{{/is_match}}
{{/is_renotify}}
```

> To send a different escalation message that do not contain the original message details, leverage a combination of `{{^is_renotify}}` and `{{#is_renotify}}` blocks:

Reviewer: ```suggestion
To send a different escalation message that does not contain the original message details, use a combination of `{{^is_renotify}}` and `{{#is_renotify}}` blocks:
```

```text
{{^is_renotify}}
This monitor is alerting and sending a first message @dev-team@company.com

To solve this monitor follow the steps:
1. Go there
2. Do this
{{/is_renotify}}

This part is generic and sent both for the first trigger and the escalation message.

{{#is_renotify}}
  This is the escalation message @dev-team@company.com
{{/is_renotify}}

```

> On monitor renotification, users will only get the following escalation message:

Reviewer: ```suggestion
On monitor renotification, users will get the following escalation message:
```
